.eslintrc

```diff
@@ -27,21 +27,38 @@
         "no-invalid-this": "error",
         "@typescript-eslint/no-throw-literal": "warn",
         "prefer-const": "error",
-        "indent": ["error", 4],
+        "@typescript-eslint/no-unused-vars": ["error",
+            {
+                "varsIgnorePattern": "^_"
+            }
+        ],
+        "indent": ["error", 4,
+            {
+                "SwitchCase": 1
+            }
+        ],
         "max-lines": ["warn", 500],
         "no-duplicate-imports": "error",
         "@typescript-eslint/array-type": "error",
         "@typescript-eslint/promise-function-async": "error",
         "no-bitwise": "error",
         "no-console": "error",
         "no-debugger": "error",
-        "prefer-template": "error",
+        "@typescript-eslint/naming-convention": ["error",
+            {
+                "selector": "default",
+                "format": ["camelCase", "PascalCase", "snake_case", "UPPER_CASE"],
+                "leadingUnderscore": "allow",
+                "trailingUnderscore": "forbid",
```

Comment:
The JSON syntax doesn't allow trailing , at the end of a block

Reply:
I think `.jsonrc` is parsed as https://json5.org/, which does allow trailing commas.

Reply:
Indeed, since it's not pure JSON, they can use a better parser... As long as it works, I'm fine with it ;)